protocols/frost/repair/repair.go

```diff
@@ -27,11 +27,6 @@ var (
 
 func Repair(helpers []party.ID, lostID, selfID party.ID, privateShare curve.Scalar) protocol.StartFunc {
 	return func(sessionID []byte) (round.Session, error) {
-		if len(helpers) < 2 {
```

Comment:
@wydengyre worth doing a `len(helpers) < 1` check? or no, I suppose that would fail in different ways

Reply:
Doesn't hurt to add it in, thanks